ArmPkg/Include/IndustryStandard/ArmFfaSvc.h

```diff
@@ -3,47 +3,170 @@
   communication between S-EL0 and the Secure Partition
   Manager(SPM)
 
-  Copyright (c) 2020, ARM Limited. All rights reserved.
+  Copyright (c) 2020-2024, Arm Limited. All rights reserved.
 
   SPDX-License-Identifier: BSD-2-Clause-Patent
 
-  @par Revision Reference:
-    - FF-A Version 1.0
+  @par Glossary:
+    - FF-A - Firmware Framework for Arm A-profile
 
+  @par Reference(s):
+    - FF-A Version 1.2 [https://developer.arm.com/documentation/den0077/latest/]
+    - FF-A Memory Management Protocol Version 1.2 [https://developer.arm.com/documentation/den0140/latest]
 
 **/
 
 #ifndef ARM_FFA_SVC_H_
 #define ARM_FFA_SVC_H_
 
-#define ARM_SVC_ID_FFA_VERSION_AARCH32               0x84000063
-#define ARM_SVC_ID_FFA_MSG_SEND_DIRECT_REQ_AARCH32   0x8400006F
-#define ARM_SVC_ID_FFA_MSG_SEND_DIRECT_RESP_AARCH32  0x84000070
-#define ARM_SVC_ID_FFA_MSG_SEND_DIRECT_REQ_AARCH64   0xC400006F
-#define ARM_SVC_ID_FFA_MSG_SEND_DIRECT_RESP_AARCH64  0xC4000070
+#define ARM_FID_FFA_ERROR            0x84000060

```

Comment:
64bit version of error function ID? It is used for partition_info_regs?

Reply:
When I see the spec Table 12.4: FFA_EREROR syntax.
FFA_ERROR has only value 0x8400060.